webpack/assets/javascripts/react_app/common/AwaitedMount.js

```diff
@@ -1,16 +1,13 @@
 import PropTypes from 'prop-types';
-import { useState, useEffect } from 'react';
+import { useState, useEffect, useCallback } from 'react';
 import store from '../redux';
 import componentRegistry from '../components/componentRegistry';
 
 // Mounts a component after all plugins have been imported to make sure that all plugins are available to the component
```

Comment:
Is this really what it does? I see it's listening for the 'loadPlugin' event now, instead of 'loadJs'.

Reply:
loadPlugin might happen after loadjs. example: user clicks a button, a tab appears, plugin has Deface::Override that calls loadPlugin